Update SSHRemoting README.md

SteveL-MSFT opened the pull request:
Changed all `-KeyPath` to `-KeyFilePath`.  Added semantic line breaks.

SteveL-MSFT committed:
Update README.md

Changed -KeyPath to -KeyFilePath.  Added semantic line breaks.

daxian-dbw commented on demos/SSHRemoting/README.md:
```diff
@@ -152,10 +168,11 @@ GitCommitId                    v6.0.0-alpha.17
-3.  Fan out to multiple machines not yet supported.
-4.  sudo command does not work in remote session to Linux machine.
+1. You can currently establish a connection either interactively with user name and password or via key authentication.  
+2. PSCredential is not yet supported.
```
It seems 1 and 2 should be together -- "_You can ... with user name and password or via key authentication,_ **but** PSCredential is not yet supported".

PaulHigin replied:
Actually it is unlikely PSCredential will be supported in the foreseeable future because it needs to be adopted by OpenSSH and there is currently much disagreement about whether it should be supported.  I feel we should just remove item 2. and not imply it is forthcoming.

daxian-dbw replied:
If item 2 is removed, then item 1 doesn't look like an issue anymore. It sounds to me like describing a feature.

PaulHigin replied:
Agreed.

SteveL-MSFT committed:
Update README.md

PaulHigin commented on demos/SSHRemoting/README.md:
```diff
@@ -1,20 +1,33 @@
+You specify the target machine with the HostName parameter and provide the user name with UserName.
+When running the cmdlets interactively at the PowerShell command line you will be prompted for a password.
+But you also have the option to use SSH key authentication and provide a private key file path with the KeyFilePath parameter.
+Note that PSCredential is not supported.
```
I wonder if we should remove this and just not mention PSCredential.  The final SSH cmdlet parameter sets don't include PSCredential so there should be no confusion.

SteveL-MSFT replied:
agree

PaulHigin commented on demos/SSHRemoting/README.md:
```diff
@@ -152,10 +168,10 @@ GitCommitId                    v6.0.0-alpha.17
-3.  Fan out to multiple machines not yet supported.
-4.  sudo command does not work in remote session to Linux machine.
+1. You can currently establish a connection either interactively with user name and password or via key authentication.  
+2. Remote debugging from Linux to Windows does not work.  However, remote debugging from Windows to Linux does work.
```
This can be removed.  Remote debugging works in both directions (actually has for a while now).

SteveL-MSFT replied:
agree

PaulHigin commented on demos/SSHRemoting/README.md:
```diff
@@ -152,10 +168,10 @@ GitCommitId                    v6.0.0-alpha.17
-4.  sudo command does not work in remote session to Linux machine.
+1. You can currently establish a connection either interactively with user name and password or via key authentication.  
+2. Remote debugging from Linux to Windows does not work.  However, remote debugging from Windows to Linux does work.
+3. Fan out to multiple machines not yet supported.
```
This can also be removed as we now support fan-out through SSH key authentication and the SSHConnection cmdlet parameter sets.

SteveL-MSFT replied:
agree

PaulHigin reviewed (changes requested):
Please look at my comments.

SteveL-MSFT committed:
Update README.md

Removed PSCredential mention as it's not supported.  Removed items from `Known Issues` that have been addressed.

PaulHigin reviewed (approved):
LGTM

daxian-dbw commented:
@SteveL-MSFT can you please resolve the Travis CI failure?

SteveL-MSFT committed:
Update .spelling

Added KeyFilePath parameter name to spelling for SSHRemoting
